lang/python/python.talon

```diff
@@ -106,3 +106,5 @@ state past: "pass"
 #^static funky <user.text>$: user.code_private_static_function(text)
 #^pro static funky <user.text>$: user.code_protected_static_function(text)
 #^pub static funky <user.text>$: user.code_public_static_function(text)
+raise {user.python_exception}: user.insert_cursor("raise {python_exception}([|])")
```

Comment:
probably something that could be moved to programming.talon / add the list to code.py? Not a deal breaker, just a thought.

Reply:
that's a reasonable thought, I can create an issue to record the desire